lib/Runtime/PlatformAgnostic/Platform/Linux/HiResTimer.cpp

```diff
@@ -0,0 +1,85 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+#include "Runtime.h"
+#include <time.h>
+#include <sys/time.h>
+#include "ChakraPlatform.h"
+
+namespace PlatformAgnostic
+{
+namespace DateTime
+{
+
+    double HiResTimer::GetSystemTime()
+    {
+// todo-osx: mach_absolute_time
+#ifndef __APPLE__
```

Comment:
I'm trying to keep current xplat effort as much compatible as possible with other platforms. IMO, that ífndef' and a comment right above would be nothing but helpful.